### PR TITLE
Explicitly set travis node versions

### PR DESCRIPTION
**Pull request by kybishop**
The default node version is ancient

### CODE REVIEW

**Comment by locks** [comment]
Isn't 0.12 still fairly used? If so, having both versions would be better.

**Comment by kybishop** [comment]
@locks I couldn't find any usage statistics, but added 0.12 regardless.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 language: node_js
+node_js:
+  - "5.2"
```
Comment (stefanpenner) [inline .travis.yml]: likely let this be 5, so it drifts with stable.

cli also supports the commonly used versions:

10
12
4
5

**Comment by kybishop** [comment]
Copied the node versions specified on ember-cli and rebased.

**Comment by locks** [comment]
Appreciated :D

**Pull request merged**